examples/algo-example/src/app/algo.nft.express.mint.example.ts

```diff
@@ -30,12 +30,12 @@ export async function algoNftExpressExample() {
   )) as TransactionHash
   console.log(`Minted nft with transaction ID: ${nftMinted.txId}`)
 
-  // fetch deployed contract address from transaction hash
-  // https://apidoc.tatum.io/tag/NFT-(ERC-721-or-compatible)#operation/NftGetContractAddress
+  // Fetch the address of the deployed smart contract from the hash of the deployment transaction.
```

Comment:
Is it relevant for NFT Express? Can a user fetch the address of the Tatum smart contract?